README.md

```diff
@@ -109,11 +109,17 @@ Rather than manually config the environment, you can also run the setup script d
 
 ### Creating Organizations and Administrators
 
-Organizations are, for example, the State of
```

Comment:
See comment above. Unless the changes pertain specifically to Minerva, it'd be great if you could move them to a separate PR.

Reply:
Yeah - it makes perfect sense to separate this out.